### DISCUSSION TITLE
Should mail processing fail if no course was detected

### DISCUSSION THREAD

**Discussion by MaxAlberti** [DHBW-SE-2023/YAAC#98] [Polls]
# Description
Currently if the mail subject doesn't contain the course, the mail parsing fails with an error. Does this make sense?
Related to Issue #86.

**Comment by MaxAlberti** [comment]
Moved from #93. 

Opinion from @leandergmeiner:
I don't think that this makes sense, because the course should not be extracted from the mail subject, as it is entered by a human and they can make mistakes.
The list already has a header which contains the course, I think we should use that instead of the mail subject.

Opinion from @V1nzent:
I would agree. Should the course always be read from the list or only if the mail subject doesn't contain it?

**Reply by leandergmeiner** [reply]
@V1nzent I think it should always read it from the list, as we are parsing that in Imgproc anyways
